src/NUnitEngine/nunit.engine.core/Services/ExtensionManager.cs

```diff
@@ -131,6 +135,24 @@ public void FindExtensions(string startDir)
                 FindExtensionsInAssembly(candidate);
         }
 
+        /// <summary>
+        /// Find and install standard extensions for a host assembly using
+        /// a built-in algorithm that searches in certain known locations.
+        /// </summary>
+        /// <param name="hostAssembly">An assembly that supports extensions.</param>
+        public void FindStandardExtensions(Assembly hostAssembly)
+        {
+            bool isChocolateyPackage = System.IO.File.Exists(Path.Combine(hostAssembly.Location, "VERIFICATION.txt"));
```

Comment:
Should this use `_fileSystem.GetDirectory`

Reply:
I did that for the FindExtensions method, but in this case it seemed unnecessarily complicated. That is, I would convert a path to an IDirectory, get an IFile from that and then convert them back to a path. But since the calling assembly must always exist, there's always a real path to start with.

Reply:
Would that argument not also cover `_startDir` which comes from the same host assembly?

Reply:
I see your point. But of course `startDir` is defined in a different method and reused multiple times, so it seems worth the trouble. Probably, it would be better to combine those two methods and just get startDir once. I'll add a TODO to my local copy to remind me to do that next time I'm in it.